### ISSUE TITLE
Rotation angle entry doesn't seem to be working

### ISSUE THREAD

**Issue by BasedOnTechnology** [LaserWeb/LaserWeb4#463]
Rotating imported files using the arrows works perfectly - however, entering an exact angle in the box above doesn't seem to do anything? I've tried both the latest build and a few builds older - all seem to have the same behavior?

Sorry if I'm missing something!

**Comment by jorgerobles** [comment]
The box is for setting the angle that will be applied upon clicking the
buttons

**Comment by BasedOnTechnology** [comment]
@jorgerobles I'm an idiot.

Sorry for the issue.

**Comment by tbfleming** [comment]
@BasedOnTechnology: no, you're not. It's not consistent with other boxes, which can lead to confusion, but I couldn't come up with a better alternative.
